tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -797,6 +797,182 @@ public function provide_test_subscription_check_detachment() {
+		$_GET['wc-stripe-hide-notice']   = 'subscription_detached';
+		$_GET['_wc_stripe_notice_nonce'] = wp_create_nonce( 'wc_stripe_hide_notices_nonce' );
+		$_REQUEST['id']                  = $subscription->get_id(); // HPOS path
+
+		$notices = $this->create_admin_notices_instance();
+		$notices->hide_notices();
+
+		$reloaded = wcs_get_subscription( $subscription->get_id() );
+		$this->assertEquals( 'yes', $reloaded->get_meta( '_wc_stripe_subscription_detached_notice_dismissed' ) );
+
+		unset( $_GET['wc-stripe-hide-notice'], $_GET['_wc_stripe_notice_nonce'], $_REQUEST['id'] );
+	}
```

PHPMD (2.15.0)

[error] test_hide_notices_dismisses_subscription_detached_notice_hpos accesses the super-global variable $_GET. (undefined)

(Superglobals)